test/test_helpers/stdout_helpers.rb

```diff
@@ -0,0 +1,34 @@
+# Taken from https://gist.github.com/searls/9caa12f66c45a72e379e7bfe4c48405b
+# Referenced in https://justin.searls.co/posts/running-rails-system-tests-with-playwright-instead-of-selenium/
```

Comment:
I haven't used it before, but maybe something like [this gem](https://github.com/Shopify/minitest-silence) might be good for this?

For some reason I thought Rails just baked in a helper to help with this but I couldn't dig up. I did find a [different approach to this here](https://gist.github.com/moertel/11091573). I don't have strong feelings about this sort of thing because it's always weird, but this other approach does avoid monkey-patching `Kernel` 🤷

Reply:
Took a peek at both of those options and I think I prefer this one because it's more conservative in two ways:

 1. It only redirects output when you explicitly call it instead of globally
 2. It only suppresses the regexp you tell it to

The aesthetic debate around whether global variable juggling or Kernel monkey patching is "less gross" might be a fun diversion, but I too don't feel that strongly and am happy to conclude by agreeing "It's always weird." 🤝 😄